api/src/test/java/org/pmiops/workbench/api/DataSetControllerTest.java

```diff
@@ -799,7 +798,7 @@ public void exportToNotebook_requiresActiveBilling() {
             workspace.getNamespace(),
             workspace.getName(),
             DbStorageEnums.workspaceActiveStatusToStorage(WorkspaceActiveStatus.ACTIVE));
-    dbWorkspace.setBillingStatus(BillingStatus.INACTIVE);
+    dbWorkspace.setInitialCreditsExhausted(true).setBillingAccountName("billingAccounts/free-tier");
```

Comment:
is there a constant we can use here instead of the string?

Reply:
Not that I'm aware of. Are you thinking a shared value for the config and the value on this line or more globally?

Reply:
maybe something like `.setBillingAccountName(BILLING_ACCOUNT_PREFIX + "/" + testFreeTier)`